lib/ansiblelint/__main__.py

```diff
@@ -31,18 +32,21 @@
 
 
 def main():
+    cwd = os.getcwd()
```

Comment:
same suggestion here
```suggestion
    cwd = pathlib.Path.cwd()
```